mm2src/lp_swap.rs

```diff
@@ -850,8 +852,70 @@ pub struct MyRecentSwapsUuids {
     pub skipped: usize,
 }
 
+#[derive(Debug)]
+pub struct MyRecentSwapsAnswer {
```

Comment:
Please use `Response` instead for uniformity. Also, it seems to me that `response` is a more common word in general for this. It's a kind of nitpick, though :slightly_smiling_face: